languages/haml/highlights.scm

```diff
@@ -8,5 +8,5 @@
   (quoted_attribute_value)
 ] @string
 
-(text_content) @string
```

Comment:
Note: I changed the node name from `text_content` to `verbatim_string` sinec this term is used in HAML.